knowledge-base/form-keep-fields-order.md

```diff
@@ -0,0 +1,105 @@
+---
+title: Keep Form Items Rendering Order after Show/Hide
+description: How to keep the form item rendering order upon toggling its visibility.
+type: how-to
+page_title: Keep Form Items Rendering Order after Show/Hide
+slug: form-kb-keep-fields-order
+position: 
+tags: 
+ticketid: 1647539
+res_type: kb
+---
+
+## Environment
+<table>
+	<tbody>
+		<tr>
+			<td>Product</td>
+			<td>Form for Blazor</td>
+		</tr>
+	</tbody>
+</table>
+
+## Description
+
+This knowledge base article answers the following questions:
+
+* How do I keep the rendering order of Form items?
+* Is it possible to toggle the visibility of a Form item and keep its rendering order instead of showing it as last?
+
+## Solution
+
+The `<FormItems>` tag doesn't render Form items directly. This tag is used only to register `<FormItem>` instances. The Form finds out about the conditional field item last, so it puts it last.
+
+Thus, the solution is to use the `TelerikFormItemRenderer` as it defines the rendering place of each form item.
```

Comment:
Would that be "instead of" or "along with"? 
```suggestion
To preserve the rendering place of each form item, you need to use the `TelerikFormItemRenderer` instead of the `<FormItems>` tag.
```

Reply:
Not technically correct so I adjusted it a little.